study.md

```diff
@@ -161,15 +169,62 @@ What is a response? what does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+The response is information sent by the server back to the client. The
+response received from google (for example) is an HTML document that the
+browser can parse and render for the user. Common status codes include:
+404 Not Found - means the requested resource is no longer available.
+403 Forbidden - means access to the resource is forbidden.
+500 Internal Server Error - a catch-all for server-side errors.
+
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+
+curl --request GET https://www.reddit.com
+
+RESULT:
+<!doctype html>
+<html>
+  <head>
+    <title>Too Many Requests</title>
+    <style>
+      body {
+          font: small verdana, arial, helvetica, sans-serif;
+          width: 600px;
+          margin: 0 auto;
+      }
+
+      h1 {
+          height: 40px;
+          background: transparent url(//www.redditstatic.com/reddit.com.header.png) no-repeat scroll top right;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>whoa there, pardner!</h1>
+
+
+
+<p>we're sorry, but you appear to be a bot and we've seen too many requests
+from you lately. we enforce a hard speed limit on requests that appear to come
+from bots to prevent abuse.</p>
+
+<p>if you are not a bot but are spoofing one via your browser's user agent
+string: please change your user agent string to avoid seeing this message
+again.</p>
+
+<p>please wait 2 second(s) and try again.</p>
+
+    <p>as a reminder to developers, we recommend that clients make no
+    more than <a href="http://github.com/reddit/reddit/wiki/API">one
+    request every two seconds</a> to avoid seeing this message.</p>
+  </body>
+</html>
+
 ```
```

Comment:
No need to put the result in there, but it does show you actually ran the command  👍